src/components/banners/NearconBanner.tsx

```diff
@@ -73,31 +53,21 @@ const Flex = styled.div<FlexProps>`
   }
 `;
 
-const Text = styled.p<TextProps>`
-  font-family: 'FK Grotesk', sans-serif;
-  font-size: ${(p) => p.size ?? '18px'};
-  line-height: ${(p) => p.lineHeight ?? '1.5'};
-  font-weight: ${(p) => p.weight ?? '400'};
-  color: ${(p) => p.color ?? '#000'};
-  margin: 0;
-`;
-
 export const NearconBanner = () => {
   const [isBannerVisible, setBanners] = useBanner();
+  const componentSrc = useCurrentComponentStore();
```

Comment:
Is it OK to use `useCurrentComponentStore` here or should I use `router` instead?